config/application.rb

```diff
@@ -159,7 +159,17 @@ class Application < Rails::Application
     end
 
     console do
-      Rails::ConsoleMethods.include(Vmdb::ConsoleMethods)
+      # This is to include vmdb methods into the top level namespace of the
+      # repl session being opened (either through `pry` or IRB)
+      #
+      # This takes a page from `pry-rails` and extends the TOPLEVEL_BINDING
+      # instead of Rails::ConsoleMethods when adding the Vmdb::ConsoleMethods.
```

Comment:
Maybe link to where this is done in pry-rails if we're "borrowing how they do it"?